packages/sdk/src/fetcher/subgraph/index.ts

```diff
@@ -468,6 +471,66 @@ class Fetcher implements IPartialCarrotFetcher {
         }
     }
 
+    private async fetchTemplateFeatureEnabledFor(
+        subgraphURL: string,
+        managerAddress: Address,
+        templateId: number,
+        featureId: number,
+        account: Address,
+    ): Promise<boolean> {
+        const lowerCaseAccount = account.toLowerCase();
+        const response = await query<GetTemplateFeatureEnabledForQueryResponse>(
+            subgraphURL,
+            GetOracleTemplateFeatureEnabledForQuery,
```

Comment:
This query should change if the manager address is the KPI token manager or the oracle manager, no?

Reply:
Oof big miss from me. Addressed in 80c567ec6b274177517a70863efa42d9b8200444